### PR TITLE
Use different libselinux version for CentOS 7/8

### PR DESCRIPTION
**Pull request by bellackn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,6 @@ env:
     - MOLECULE_DISTRO: centos6
     - MOLECULE_DISTRO: ubuntu1804
     - MOLECULE_DISTRO: ubuntu1604
```
Comment (tobias-richter) [inline .travis.yml:14]: why are we not adding centos8 here as a distro?

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@ provisioner:
   name: ansible
   lint:
     name: ansible-lint
+    options:
```
Comment (tobias-richter) [inline molecule/default/molecule.yml]: This should no longer be necessary, correct? I can not find any skip tag in the changes.

File: tasks/configure-RedHat.yml [hunk 3]
```diff
@@ -65,24 +81,10 @@
   when:
     - ansible_distribution_major_version | int >= 8
     - not localhost_cert.stat.exists
+
```
Comment (tobias-richter) [inline tasks/configure-RedHat.yml]: Unrelated whitespace change.

**Review by tobias-richter — CHANGES_REQUESTED** [review]
Looks good, only nitpicks.

File: molecule/default/molecule.yml (hunk 2 above)
Comment (bellackn) [reply molecule/default/molecule.yml]: Correct, will be deleted.

File: .travis.yml (hunk 1 above)
Comment (bellackn) [reply .travis.yml:14]: Oh, I didn't assume geerlingguy has made a Docker image for CentOS 8, but he actually has. I will add this.

File: tasks/configure-RedHat.yml (hunk 3 above)
Comment (bellackn) [reply tasks/configure-RedHat.yml]: I would stick with this newline, it seems like this was a formatting error in the first place. If you insist, I will delete it 😛

Comment (tobias-richter) [reply tasks/configure-RedHat.yml]: Ok :)

**Review by tobias-richter — APPROVED** [review]
